shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/networking/OpenshiftNetworkingService.java

```diff
@@ -59,6 +60,7 @@ private Route buildRoute(BridgeIngress bridgeIngress, Service service) {
 
         // We have to provide the host manually in order not to exceed the 63 char limit in the dns label https://issues.redhat.com/browse/MGDOBR-271
         route.getSpec().setHost(String.format("%s.%s", bridgeIngress.getMetadata().getName(), getOpenshiftAppsDomain()));
+        route.getSpec().setPath(EVENTS_ENDPOINT_SUFFIX);
```

Comment:
The similar change for OpenShift has me more puzzled.. I _guess_ at this; but AFAIK have no where to test it.